<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnDcMatrix">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCitraStegano">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCitra1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Citra 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="citraSteganoPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKembali">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kembali</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCitra2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Citra 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dcMatrixPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMSE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPSNR">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHitungMSE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hitung MSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHitungPSNR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hitung PSNR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMSE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MSE</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPSNR">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PSNR</string>
   </property>
  </widget>
  <zorder>btnDcMatrix</zorder>
  <zorder>btnCitraStegano</zorder>
  <zorder>lblCitra1</zorder>
  <zorder>citraSteganoPath</zorder>
  <zorder>graphicsView1</zorder>
  <zorder>btnKembali</zorder>
  <zorder>lblCitra2</zorder>
  <zorder>dcMatrixPath</zorder>
  <zorder>graphicsView2</zorder>
  <zorder>txtMSE</zorder>
  <zorder>txtPSNR</zorder>
  <zorder>btnHitungMSE</zorder>
  <zorder>btnHitungPSNR</zorder>
  <zorder>lblPSNR</zorder>
  <zorder>lblMSE</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
